modules/trading.py

```diff
@@ -136,7 +122,7 @@ def cycle(self):
                                         cargo.SetQuantity(quant)
                                         price = prod[1]+vsrandom.uniform(-1,1)*prod[2]
                                         cargo.SetPrice(cargo.GetPrice()*price)
-                                        debug.debug("Adding %s of %s cargo for %s", quant, cargo.GetContent(), price)
+                                        debug.debug("Adding "+str(quant)+" of "+cargo.GetContent()+" cargo for "+str(price))
```

Comment:
Looking at the debug module (which we own), then the proper fix should be:
```python
debug.debug(
    "Adding %d of %s for %f", quant. cargo.GetContent(), price)
)
```

Reply:
@BenjamenMeyer I'm a little lost with the modules/trading.py comment. 

I trust that you know the right thing to do, but I don't know what it is that you want me to do with the information. 

Is this something to be fixed now, in this PR? If it is, how is THAT done? Thanks so much for your patience while I learn the ropes. 

Or, is this something that we'll have to fix with a seperate PR once this one is merged?

Reply:
How do I fix this? I do not have a lot of experience with PR yet.

Reply:
Since you're reverting we can do it in another PR. Just noting what the correct printf args would be for the parameters.